projects/Amlogic-ng/initramfs/platform_init

```diff
@@ -51,9 +54,15 @@ case $display_mode in
   480*)            X=720  Y=480  ;;
   576*)            X=720  Y=576  ;;
   720p*)           X=1280 Y=720  ;;
+  2160p*)          X=3840 Y=2160 ;;
   *)               X=1920 Y=1080 ;;
 esac
 
+if [ "$native_4k_gui" == "0" ]; then
```

Comment:
I think you should only fallback to 1080p if the current X or Y correspond with a higher resolution.

Reply:
Ohh right. Bug